File: ThreeNo_Multiplication/Mult.js
/*
    Author : Pakshal Jain
    Date : 12/10/2022
    Print : To Calulate Multiplication of Three Numbers
*/
//Function To Call Mult Function
function Mult(No, No2, No3) {
    var ans = 0;
    //Handeling Conditions if Any of Entered Number is Zero
    if ((No == 0) && (No2 == 0)) {
        ans = No3;
    }
    else if ((No2 == 0) && (No3 == 0)) {
        ans = No;
    }
    else if ((No == 0) && (No3 == 0)) {
        ans = No2;
    }
    else if (No == 0) {
        ans = No2 * No3;
    }
    else if (No2 == 0) {
        ans = No * No3;
    }
    else if (No3 == 0) {
        ans = No * No3;
    }
    //Handeling Condition if Any of Entered Number is Not Zero
    else {
        ans = No * No2 * No3;
    }
    return ans;
}
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
function main() {
    //Requried variables
    var No = 1;
    var No2 = 1;
    var No3 = 2;
    var iret = 0;
    console.log("Jay Ganesh....");
    //Handeling Condtion if All Entered Numbers are Zero
    if ((No == 0) && (No2 == 0) && (No3 == 0)) {
        iret = No * No2 * No3;
    }
    else {
        //Call to Mult Function
        iret = Mult(No, No2, No3);
    }
    //Printing Final Output
    console.log("Multiplication of Three Entered Number is : %d", iret);
}
//Call To Main Function
main();

File: ThreeNo_Multiplication/Mult.go
/*
	Author : Pakshal Jain
	Date : 12/10/2022
	Program : To Calulate Multiplication of Entered Three Numbers
*/

//Required Libraries
package main
import "fmt";

//Function To Calculate Multiplcation
func Mult(No int,No2 int,No3 int) int {
	var ans int = 0;

	//Conditions To Handel if any Of Entered Number is Zero
	if No == 0 && No2 == 0 {
		ans = No3;
	} else if No2 == 0 && No3 == 0 {
		ans = No;
	} else if No == 0 && No3 == 0 {
		ans = No2;
	} else if No == 0 {
		ans = No2 * No3;
	} else if No2 == 0 {
		ans = No * No3;
	} else if No3 == 0 {
		ans = No * No2;
	} else {          				//Handeling Condtion if Non Of The Entered Number is Zero                   
		ans = No * No2 * No3;
	}

	return ans;
}

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

func main() {
	//Requried Variables
	var No int = 0;
	var No2 int = 0;
	var No3 int = 0;
	var iret int = 0;

	fmt.Println("Jay Ganesh....");
	
	fmt.Println("Enter First Number");
	fmt.Scan(&No);

	fmt.Println("Enter Second Number");
	fmt.Scan(&No2);

	fmt.Println("Enter Third Number");
	fmt.Scan(&No3);

	//Handeling Condition all The Entered Numbers are Zero
	if No == 0 && No2 == 0 && No3 == 0 {
		iret = No * No2 * No3;
	} else {
		//Calling Mult Function
		iret = Mult(No,No2,No3);
	}

	//Printing final Output
	fmt.Printf("Multiplication of Entered Three Numbers is : %d",iret);
}